nixos/tests/switch-test.nix

```diff
@@ -699,11 +811,11 @@ import ./make-test-python.nix ({ pkgs, ...} : {
         assert_contains(out, "OnCalendar=2014-03-25 02:59:56 UTC")
         out = switch_to_specialisation("${machine}", "timerModified")
         assert_lacks(out, "stopping the following units:")
+        assert_lacks(out, "NOT restarting the following units:")
         assert_lacks(out, "reloading the following units:")
-        assert_contains(out, "restarting the following units: test-timer.timer\n")
+        assert_contains(out, "\nrestarting the following units: test-timer.timer\n")
```

Comment:
why aren't we stripping away the spaces from each `out` variable (or rather in the `switch_to_specialisation` function)? is it important to check the newline everywhere?

Reply:
Because `restarting the following units` and `starting the following units` must somehow be differentiated and using regex would be pretty expensive to run hundreds of times

Reply:
ah, i didn't know `out` was a string, thought it was an array, my bad